### PR TITLE
doc: Fix usage of its/it's

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

File: eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/design/checkstyle-metadata.properties [hunk 1]
```diff
@@ -30,7 +30,7 @@ ForbidWildcardAsReturnTypeCheck.allowReturnWildcardWithExtends = Allow wildcard
...
...href='http://docs.oracle.com/javase/tutorial/java/javaOO/nested.html'>private inner classes</a>, interfaces or enumerations.<br><br>Examples:<code><pre> class OuterClass {<br><br>  public InnerClass innerFromMain = new InnerClass(); //WARNING<br>  private class InnerClass { ... }<br>  public InnerClass  getValue() { //WARNING<br>      return new InnerClass();<br>  }<br><br>  private interface InnerInterface { ... }<br>  public Set&ltInnerInterface&gt getValue() { //WARNING<br>      return new TreeSet&ltInnerInterface&gt;;<br>  }<br><br>  private Enum Fruit {Apple, Pear}<br>  public Fruit getValue() { //WARNING<br>      return Fruit.Apple;<br>  }<br><br>  public someMethod(InnerClass innerClass) { ... }  //WARNING<br><br> }</pre></code><b>Rationale:</b> it is possible to return<br>private inner type or use it as the parameter of non-private method, but it is impossible<br>to use it in other classes (besides inner classes)<br>unless it extends or implements at least one <u>non-private</u> class or interface.<br>Such situation usually happens after bulk refactoring and usually means dead/useless code<br></p><br>@author <a href="mailto:nesterenko-aleksey@list.ru">Aleksey Nesterenko</a>
```
Comment (romani) [inline eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/design/checkstyle-metadata.properties]: image

**Comment by romani** [comment]
@pzygielo , please rebase on our latest master to fix CI.

**Comment by rnveach** [comment]
Wercker failure is failure to download sevntu-checkstyle-maven-plugin

**Comment by pzygielo** [comment]
> @pzygielo , please rebase on our latest master to fix CI.

Rebased and force-pushed. Didn't help for wercker Struts step with the same failure after:
> [WARNING] The POM for com.github.sevntu-checkstyle:sevntu-checkstyle-maven-plugin:jar:1.34.1 is missing, no dependency information available

**Comment by pzygielo** [comment]
Unfortunately other green werckers have that step reporting
> build is skipped ...
So I myself can't deduce what is wrong.

**Comment by rnveach** [comment]
@romani 1.34.1 for maven sevntu plugin hasn't been released yet.
https://mvnrepository.com/artifact/com.github.sevntu-checkstyle/sevntu-checkstyle-maven-plugin

Is there a delay?

**Comment by pzygielo** [comment]
> @romani 1.34.1 for maven sevntu plugin hasn't been released yet.
> https://mvnrepository.com/artifact/com.github.sevntu-checkstyle/sevntu-checkstyle-maven-plugin
> 
> Is there a delay?

It is in central I think: https://repo1.maven.org/maven2/com/github/sevntu-checkstyle/sevntu-checkstyle-maven-plugin/1.34.1

**Comment by romani** [comment]
Web site for artifacts use search on index, it takes some time to catch-up for it.

I always looks at https://repo1.maven.org/maven2/com/github/sevntu-checkstyle/ , link is readme.
All artifacts are there.
I did clear for wercker cache

**Comment by rnveach** [comment]
Yea, I see everything in maven central, weird.

**Pull request merged**